### DISCUSSION TITLE
PUT/DELETE method for dcat of the server requires {CatalogId}

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#711] [Ideas]
![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/637c6231-e6e2-40af-8446-716b21fcadf4)

By http://localhost:8080/v1/swagger-ui/index.html?urls.primaryName=admin#/.
PUT/DELETE method requires {catalogId} for deleteing/updating the metadata of the server.
For now there is no way to retrieve catalogId, I think it might be good to have an API call or somewhere to find the info about catalogId. 
Thanks.

**Comment by Tomvbe** [comment]
Yes, I had the same frustration. I was going to suggest a solution for this this sprint as well. Since we only support one catalog on a server, I wonder if we can't just hide the id from the API.

**Comment by Tomvbe** [comment]
Discussed on dev session, for us it is ok to implement this change, @ddvlanck do you agree with this?
The server can only ever have one catalog so we would remove the path parameter for catalogId.
